src/components/pages/roster-page/roster-page.tsx

```diff
@@ -370,152 +372,157 @@ export const RosterPage = () => {
       <Container maxWidth="md">
         <PageHeader title="Roster" />
 
```

Comment:
Not sure why the GitHub diff engine gets so confused on these minor changes - the diff is much simpler in VS Code. All I did was insert the ```<ErrorBoundary>``` component into the markup which changed the indentation of the markup it wraps...